roxie/udplib/udptrs.cpp

```diff
@@ -258,8 +265,9 @@ class UdpReceiverEntry : public IUdpReceiverEntry
 
     inline void pushData(unsigned queue, DataBuffer *buffer)
     {
-        output_queue[queue].pushOwn(buffer);
-        if (!packetsQueued++)
+        int numActive = output_queue[queue].pushOwn(buffer);
+        packetsQueued++;
+        if ((numActive % udpPacketsRTSFactor) == 0)
```

Comment:
Previously it sent a request if it was the first packet being queued on any queue, now it sends it for the first on each queue, and every 100?
This could do with commenting why this is better/what problem it solves.

Reply:
In testing I seemed to find times when packets were in the queue and not being sent and I wanted a knob to see if we could request to send more often, perhaps the 100 should be maxSlotsPerSender.